templates/travis/.travis/install.sh.j2

```diff
@@ -109,7 +109,11 @@ spec:
     username: pulp
     password: pulp
     admin_password: pulp
-  content_host: $(hostname):24816
```

Comment:
So we will not set this for all plugins anymore? It is a temporary measure, but still.